sdk/lib/core/uri.dart

```diff
@@ -512,21 +554,38 @@ abstract interface class Uri {
 
   /// The fragment identifier component.
   ///
+  /// A fragment identifier is a string after URI, after the hash, which
+  /// identifies something specific as a function of the document. For
+  /// a user interface Web document such as HTML page, it traditionally
+  /// identifies a section or part of the page. In modern pages, it can
+  /// indicate a particular view of the page.
+  ///
   /// The value is the empty string if there is no fragment identifier
   /// component.
   String get fragment;
 
-  /// The URI path split into its segments.
+  /// Same as the URI [path], but split into its segments.
```

Comment:
So it's not the *same*. I'd use the original phrasing, but add the `[path]` link.